spec/components/email/receiver_spec.rb

```diff
@@ -985,6 +985,93 @@ def create_post_reply_key(value)
       end
     end
 
+    context "emailing a group by email_username and following reply flow" do
+      let!(:topic) do
+        group.update(email_username: "team@somesmtpaddress.com")
+        process(:email_to_group_email_username_1)
+        Topic.last
+      end
+      let(:group_user) do
```

Comment:
I think the naming is confusing here because `group_user` it is the name for the join table as well. Maybe just `user` or `user_with_group`?

Reply:
Ah good point, changing